vuepress/docs/next/docs/getting-started/README.md

```diff
@@ -173,106 +171,80 @@ sudo kubectl create namespace entando
+If you run into network issues during startup, or if you are using Windows for your local development instance, please refer to [network issues](../reference/local-tips-and-tricks.md#network-issues). Symptoms can include Entando failing to completely start for the first time or a working Entando instance failing to restart later.
 :::
 
-To set up external access to your cluster, you'll need to replace the value of
-`entando.default.routing.suffix` with your Ubuntu IP.
+To set up external access to your cluster, you need to specify the fully qualified domain of your Ubuntu VM. This follows the pattern YOUR-INSTANCE.YOUR-HOST-IP.nip.io. 
+
+ON Mac and Ubuntu OS, YOUR-INSTANCE is `quickstart`. On Windows with Hyper-V, YOUR-INSTANCE is the name of your VM followed by `.mshome.net`, e.g. `entando.mshome.net`. 
+
+Find your IP with the next step, then concatenate your values for YOUR-INSTANCE and YOUR-HOST-IP with `.nip.io` to create the domain name, e.g. `entando.mshome.net.192.168.64.33.nip.io`.
 
-1. Get the IP address of your Ubuntu VM
+1. Return the IP address of your Ubuntu VM.
 
 ``` bash
 hostname -I | awk '{print $1}'
 ```
 
-2. Save that value for the deployment steps below
-
-### Deploy Entando
-
-To speed up the _Getting Started_ environment, embedded databases are used by default for these components.
-See [this tutorial](../../tutorials/devops/default-database.md) for more information on how to change your
-database connection.
-
+2. Download the file `entandoapp.yaml`.
 
-1. Enter the Helm quickstart directory
-
-``` bash
-cd entando-helm-quickstart-6.3.2
+```
+curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/samples/entando-app.yaml"
 ```
 
-2. Edit the file in `sample-configmaps/entando-operator-config.yaml` by uncommenting the value for `entando.default.routing.suffix:` and setting the value to the IP address of your Ubuntu VM plus `.nip.io`. For example, `entando.default.routing.suffix: 192.168.64.21.nip.io`. Pay attention to yaml spacing.
+3. Modify `entandoapp.yaml` to reflect the following: 
 
-Reduced network speed or Docker Hub traffic controls may cause the Entando install to timeout. Use the ConfigMap settings below to increase the thresholds for image downloads.
+- `metadata name`: quickstart (Mac/Ubuntu) or the name of your VM (Windows)
+- `dbms`: embedded
+- `ingressHostName`: YOUR-INSTANCE.YOUR-HOST-IP.nip.io
+- `standardServerImage`: wildfly
 
-``` bash
-entando.pod.completion.timeout.seconds: "1200"
-entando.pod.readiness.timeout.seconds: "1200"
-```
 
-3. Deploy your config map
+::: tip Embedded Databases
+To speed up the _Getting Started_ environment, embedded databases are used by default for these components.
+See [this tutorial](../../tutorials/devops/default-database.md) for more information on how to change your
```

Comment:
See [this tutorial](../../tutorials/devops/default-database.md)>> See the [Tutorial on Default Databases](../../tutorials/devops/default-database.md) for more ...

Reply:
good catch